x-pack/legacy/plugins/TEST_PLAN.md

```diff
@@ -0,0 +1,20 @@
+# Test plan process (7.4.0)
+
+### What
+Everything that is being released in the 7.4 should be tested by someone other than the person who implemented it 
+
+### When
+The first week after feature freeze (FF) all known bugs and blockers will be fixed. The test plan will begin shortly after this (typically one week after FF date)
+
+### Prerequesites
+ - Every pull request that is merged to 7.x (or backported to 7.4 after FF) must be given the label: `7.4.0`. [These pull requests](https://github.com/elastic/kibana/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3Av7.4.0+label%3ATeam%3Aapm+-label%3Abackport) will provide the basis of our test plan. 
+ - Immediately before starting the test plan all relevant PRs for the release are selected and the label `in-test-plan` is applied. The "assigned" field will be reset to nobody (read further why)
+
+### Execution
+ - [Find the list](https://github.com/elastic/kibana/pulls?q=is%3Apr+label%3Av7.4.0+label%3Ain-test-plan+label%3ATeam%3Aapm+-label%3Atested-after-ff+no%3Aassignee) of unassigned, untested issues in the test plan
+ - Assign yourself to an issue as you start testing it to avoid duplicate work
+ - Testing a PR can have two outcomes:
+   - It worked as expected => Apply `tested-after-ff` label
+   - It did not work as expected => Create a bug report and link to it from the PR
```

Comment:
should the bug have the label applied as well?  The original issue's label should only be added after the bug is resolved.

Reply:
I think the issue created for the bug should have the version label (in this case `v7.4.0`). But I don't think it makes sense to add `in-test-plan` since that's only for PRs that have already been merged (and can actually be tested)

Reply:
With regards to the process when a bug is discovered I'm leaning towards still marking the PR as tested with `tested-after-ff`. Since an issue will be created (which should have high priority) it will eventually lead to a new PR which will end up in the test plan. And having duplicate PRs in the test plan is maybe confusing.